Lets Do This/Controllers/Campaign/LDTCampaignDetailViewController.m

```diff
@@ -209,6 +209,15 @@ - (void)configureReportbackItemCell:(LDTCampaignDetailReportbackItemCell *)repor
     reportbackItemDetailView.userDisplayNameButtonTitle = reportbackItem.user.displayName;
 }
 
+- (CGSize)reportbackItemCellSizeForIndexPath:(NSIndexPath *)indexPath {
+    UINib *reportbackItemCellNib = [UINib nibWithNibName:@"LDTCampaignDetailReportbackItemCell" bundle:nil];
+    LDTCampaignDetailReportbackItemCell *sizingCell = [[reportbackItemCellNib instantiateWithOwner:nil options:nil] firstObject];
```

Comment:
Storing as a property still doesn't sound like a bad idea though: https://github.com/honghaoz/Dynamic-Collection-View-Cell-With-Auto-Layout-Demo#view-controller-of-uicollectionview

> Since `collectionView:layout:sizeForItemAtIndexPath:` is called before `cellForItemAtIndexPath:`, so we need to initialize a cell and let system use auto layout to calculate height for us. To avoid memory leak, we use a dictionary to cache the cells that are off screen (not shown on screen)